schematic/store/synapse.py

```diff
@@ -762,7 +762,10 @@ def uplodad_manifest_file(self, manifest, metadataManifestPath, datasetId, restr
         # update file name
         file_name_full = metadataManifestPath.split('/')[-1]
         file_extension = file_name_full.split('.')[-1]
-        file_name_new = 'synapse_storage_manifest_' + component_name + '.' + file_extension
+        if restrict_manifest: 
```

Comment:
The logic for naming should be updated. Currently, the censored manifest does not have the `censored` appended, the uncensored manifest does. Restrictions are correctly generated for the uncensored manifest, naming is just off.

Reply:
I think it is a bit confusing to use `restrict_manifest` to infer if a manifest is a censored manifest or not. Let me switch it to use the file name.

Reply:
Yep! It's based off the file name elsewhere as well